geode-docs/topologies_and_comm/multi_site_configuration/setting_up_a_multisite_system.html.md.erb

```diff
@@ -310,81 +310,79 @@ As an example, assume the following scenario:
+gfsh -e "connect --locator=localhost[10331]" -e "list gateways"
+...
+(2) Executing - list gateways
 
-    GatewaySender Id |              Member               | Remote Cluster Id |   Type   | Status  | Queued Events | Receiver Location
-    ---------------- | --------------------------------- | ----------------- | -------- | ------- | ------------- | -----------------
-    ln               | boglesbymac(ny-1:88641)<v2>:33491 | 2                 | Parallel | Running | 0             | boglesbymac:5037
-    ln               | boglesbymac(ny-2:88705)<v3>:29329 | 2                 | Parallel | Running | 0             | boglesbymac:5064
-    ln               | boglesbymac(ny-3:88715)<v4>:36808 | 2                 | Parallel | Running | 0             | boglesbymac:5132
-    ln               | boglesbymac(ny-4:88724)<v5>:52993 | 2                 | Parallel | Running | 0             | boglesbymac:5324
+GatewaySender Section
 
-    GatewayReceiver
+GatewaySender Id |              Member               | Remote Cluster Id |   Type   | Status  | Queued Events | Receiver Location
+---------------- | --------------------------------- | ----------------- | -------- | ------- | ------------- | -----------------
+ln               | mymac(ny-1:88641)<v2>:33491       | 2                 | Parallel | Running | 0             | mymac:5037
+ln               | mymac(ny-2:88705)<v3>:29329       | 2                 | Parallel | Running | 0             | mymac:5064
+ln               | mymac(ny-3:88715)<v4>:36808       | 2                 | Parallel | Running | 0             | mymac:5132
+ln               | mymac(ny-4:88724)<v5>:52993       | 2                 | Parallel | Running | 0             | mymac:5324
 
-                 Member               | Port | Sender Count | Sender's Connected
-    --------------------------------- | ---- | ------------ | --------------------------------------------------------------------------
-    boglesbymac(ny-1:88641)<v2>:33491 | 5057 | 24           | ["boglesbymac(ln-1:88651)<v2>:48277","boglesbymac(ln-4:88681)<v5>:42784","boglesbymac(ln-2:88662)<v3>:12796","boglesbymac(ln-3:88672)<v4>:43675"]
-    boglesbymac(ny-2:88705)<v3>:29329 | 5082 | 0            | []
-    boglesbymac(ny-3:88715)<v4>:36808 | 5371 | 0            | []
-    boglesbymac(ny-4:88724)<v5>:52993 | 5247 | 0            | []
+GatewayReceiver Section
 
+             Member               | Port | Sender Count | Senders Connected
+--------------------------------- | ---- | ------------ | --------------------------------------------------------------------------
+mymac(ny-1:88641)<v2>:33491       | 5057 | 24           | ["mymac(ln-1:88651)<v2>:48277","mymac(ln-4:88681)<v5>:42784","mymac(ln-2:88662)<v3>:12796","mymac(ln-3:88672)<v4>:43675"]
+mymac(ny-2:88705)<v3>:29329       | 5082 | 0            | []
+mymac(ny-3:88715)<v4>:36808       | 5371 | 0            | []
+mymac(ny-4:88724)<v5>:52993       | 5247 | 0            | []
+```
 
 Execute the load-balance command:
+``` pre
```

Comment:
One format correction: please insert a blank line following "Execute the load-balance command:" so Markdown will properly recognize the first line of the example code. Thanks!

Reply:
Done, thanks for the review!